Framework/Geometry/test/ShapeInfoTest.h

```diff
@@ -122,6 +155,33 @@ class ShapeInfoTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(tshape, ShapeInfo::GeometryShape::SPHERE);
   }
 
+  void testGetObjectGeometryForCylinders() {
+    ShapeInfo shapeInfo;
+    V3D centreBottomBase(0, 0, 0);
+    V3D symmetryAxis(0, 1, 1);
+    double innerRadius = 1;
+    double outerRadius = 2;
+    double height = 5;
+    shapeInfo.setHollowCylinder(centreBottomBase, symmetryAxis, innerRadius,
+                                outerRadius, height);
+
+    ShapeInfo::GeometryShape tshape;
+    std::vector<V3D> tpoints;
+    double theight;
```

Comment:
Could you use some more verbose variable names here?

e.g. just expanding `t` to `test`.